sdk-extensions/tracing-incubator/src/jmh/java/io/opentelemetry/sdk/extension/incubator/trace/ExecutorServiceSpanProcessorCpuBenchmark.java

```diff
@@ -38,7 +38,7 @@
 
   @State(Scope.Benchmark)
   public static class BenchmarkState {
-    private MetricProducer collector;
+    private InMemoryMetricReader collector;
```

Comment:
Nit: I don't think we should call these variables `collector`, since that's doubly overloaded with prometheus and the otel collector.

Reply:
Tried to fix this across all these benchmarks.  I'm thinking there's a huge opportunity for some code sharing in these Span-processor benchmarks, but perhaps in a future PR.